### ISSUE TITLE
2 - Get the CMS system installed

### ISSUE THREAD

**Issue by burlexpo** [bethlakshmi/GBE2#398]
Install the CMS system, test it, set up any templates needed so Scratch can start creating static pages.

**Comment by burlexpo** [comment]
I emailed this to @jonkiparsky and @bethlakshmi on 4/5 -- no commentary yet.  I'm hoping that posting it in here will inspire some action...

Using https://code.djangoproject.com/wiki/CMSAppsComparison as a jumping off point, I narrowed it down to *Merengue* (http://www.merengueproject.org/), *Django-CMS* (https://www.django-cms.org/en/), and *Mezzanine* (http://mezzanine.jupo.org/).

I've looked at all of them and they all have advantages and disadvantages.  *Mezzanine* has themes available, but no add-ons (as far as I can tell).

*Django-CMS* can use themes, but doesn't seem to have any pre-created ones available for download, but it has a ton of plugins to easily extend its functionality.

*Merengue* is halfway between the two.

So, now that I've narrowed it down to three, I'd love some technical evaluations...
